bc_obps/reporting/migrations/0007_prod_data.py

```diff
@@ -614,6 +614,18 @@ def reverse_init_methodology_data(apps, schema_monitor):
     ).delete()
 
 
+def load_reporting_fixtures(apps, schema_editor):
+    from django.core.management import call_command
+
+    fixture_files = [
+        'reporting/fixtures/mock/reporting_year.json',
+    ]
+
+    # Load the fixtures
+    for fixture in fixture_files:
+        call_command('loaddata', fixture)
+
+
```

Comment:
Sorry about the nitpicks! This is missing a reverse function, and we can name it after the reporting year
```
def init_reporting_years(..):
```